osv/impact.py

```diff
@@ -91,7 +91,9 @@ def get_affected(self,
                    fix_commits,
                    limit_commits=None):
     """"Get list of affected tags and commits for a bug given regressed and
-    fixed commits."""
+
```

Comment:
unintentional change?

Reply:
Ah... I set up a git hook to run pyformat on all files committed. Not sure why pyformat likes adding blank lines to multiline doc comments though